### ISSUE TITLE
On the events listing I’d like an "add to calendar feature " or even better an RSVP feature.

### ISSUE THREAD

**Issue by davidsilvasmith** [brightdigit/lansingcodes-apple#12]

**Comment by leogdion** [comment]
The best way to make sure the workflow facilitates RSVPing on whatever platform the event is hosted on. Most platforms (i.e. Meetup) should send calendar events via email when you RSVP.

**Comment by leogdion** [comment]
@egillespie Is there a way to do this with the meetup api?

**Comment by egillespie** [comment]
It looks like an RSVP action on Meetup.com results in the following HTTP request:

```
POST https://www.meetup.com/mu_api/urlname/events/eventId
Form data:
queries: (endpoint:Code-for-Lansing/events/fpfsmrybcdbqb/rsvps,meta:(method:post),params:(eventId:fpfsmrybcdbqb,fields:rsvp_counts,response:yes,urlname:Code-for-Lansing),ref:rsvpAction_Code-for-Lansing_fpfsmrybcdbqb)
```

In this example, `Code-for-Lansing` is the group key and `fpfsmrybcdbqb` is the event identifier. It may be possible for us to recreate this request, but there may be CORS or Referer protection to prevent this (we won't know until we try).

Also, the particular event identifier (`fpfsmrybcdbqb`) is one of two identifiers that Meetup uses for events. The other is numeric and appears in URLs (`268401713` in this particular example). The numeric identifier is the one exposed through Meetup's iCal exports and the one we have available in the Lansing Codes database. If the RSVP does not accept the numeric identifier, we may not have a way to simplify the RSVP process.

---

All this being said, we can always recreate iCal events to let people add events to their calendar without RSVPing. This would obviously make it more difficult to ensure enough space and food, and be more difficult to contact attendees before and after our events.

**Comment by leogdion** [comment]
Yeah I'd want RSVP-ing part of the workflow in order to get the calendar event. I'll look into it further.
